Source/PatchingUtilities.cs

```diff
@@ -314,5 +317,189 @@ public static void InitCancelOnUI()
         private static bool CancelDuringAlerts() => !ShouldCancel;
 
         #endregion
+
+        #region Find.CurrentMap replacer
+
+        public static void ReplaceCurrentMapUsage(MethodBase method)
+        {
+            if (method != null)
```

Comment:
It should write a warning at least but I don't mind if it errors, methods disappear sometimes when mods update. The more attention is drawn to it the likehood for it to be reported to us is greater so we can fix it. If it's quiet, we will fail to notice.

Reply:
Should now be handled